test/bench_components_mlkem.c

```diff
@@ -199,63 +199,43 @@ static int bench(void)
         mlk_gen_matrix((mlk_polyvec *)data0, (uint8_t *)data1, 0))
 
 
+#if defined(MLK_ARITH_BACKEND_AARCH64_CLEAN) || \
```

Comment:
As discussed offline, can we just get rid of the distinction between MLK_ARITH_BACKEND_AARCH64_CLEAN and MLK_ARITH_BACKEND_AARCH64_OPT?

Reply:
Yes. Done.

Reply:
Super, looking great now